workshops/intro_to_ML/README.md

```diff
@@ -0,0 +1,272 @@
+---
+name: 'Intro To ML'
+description: 'Making a Machine Learning model to predict relation between 2 number sets '
+author: '@Agrim-Bansal'
+---
+
+
+# Introduction
```

Comment:
please make this the title of the workshop

Reply:
Could you please elaborate a bit? I couldn't understand

Reply:
@sampoder

Reply:
just make it # Intro to ML

Reply:
just so that the site works